.travis.yml

```diff
@@ -3,5 +3,5 @@ node_js:
   - "node"
 notifications:
   email: false
-script: yarn test --ci
+script: yarn test --ci && yarn run prettier:ci --ci
```

Comment:
This is the only part I'm not sure about.

I think it's good to run prettier during CI, but is it better to make it part of the test command or chain two commands like this?

Reply:
Let's put it in the test script, since any failure on CI should probably be reflected locally too.